catalog/service/rest/src/main/java/org/projectnessie/catalog/service/rest/ExternalBaseUri.java

```diff
@@ -91,11 +91,13 @@ default Map<String, String> icebergConfigOverrides() {
         "uri", icebergBaseURI().toString());
   }
 
-  default URI icebergS3SignerUri(String prefix, ContentKey contentKey) {
+  default URI icebergS3SignerUri(String prefix, ContentKey contentKey, String encryptedLocation) {
```

Comment:
Same here, let's remove the new parameter for now.

Reply:
We can't, we need it.

Reply:
For staged tables.

Reply:
Yes - `location` but not "encrypted".

Reply:
(Meant "encrypted" with "remove" - my bad)